flink-formats/flink-orc/src/test/java/org/apache/flink/orc/writer/OrcBulkWriterTest.java

```diff
@@ -46,16 +57,8 @@ public class OrcBulkWriterTest {
     private final List<Record> input =
             Arrays.asList(new Record("Shiv", 44), new Record("Jesse", 23), new Record("Walt", 50));
 
-    @Test
-    public void testOrcBulkWriter() throws Exception {
-        final File outDir = TEMPORARY_FOLDER.newFolder();
-        final Properties writerProps = new Properties();
-        writerProps.setProperty("orc.compress", "LZ4");
-
-        final OrcBulkWriterFactory<Record> writer =
-                new OrcBulkWriterFactory<>(
-                        new RecordVectorizer(schema), writerProps, new Configuration());
-
+    private void writeRecordsIntoOrcFile(File outDir, OrcBulkWriterFactory<Record> writer)
```

Comment:
nit: Can we move this method after the ones using them?

Reply:
done